### PR TITLE
Add withdrawing an offer entry

### PR DESCRIPTION
**Pull request by adamsilver**
Add screenshots, user needs and research findings with actions.

### CODE REVIEW

File: app/posts/manage-teacher-training-applications/2020-01-31-withdrawing-an-offer.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+title: Withdrawing an offer
+description: Flow for withdrawing an offer
+date: 2020-01-31
+---
+
+This flow shows how a provider can withdraw their offer.
+
+## User needs
+
+As a user
```
Comment (fofr) [inline app/posts/manage-teacher-training-applications/2020-01-31-withdrawing-an-offer.md]:
```suggestion
As a provider
```

File: app/posts/manage-teacher-training-applications/2020-01-31-withdrawing-an-offer.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+I need to withdraw an offer
+So that I can rectify a mistake I've made or stop an unsuitable candidate from taking a spot
+
+## Research findings
+
+1 participant said to change from ‘Withdrawn by us’ to ‘Offer withdrawn’ because a candidate cannot withdraw an offer, only their application. We'll change the states for the next round of research to:
+
+- ‘Offer withdrawn’
```
Comment (fofr) [inline app/posts/manage-teacher-training-applications/2020-01-31-withdrawing-an-offer.md]:
```suggestion
- Offer withdrawn
```

File: app/posts/manage-teacher-training-applications/2020-01-31-withdrawing-an-offer.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+- ‘Application withdrawn’
```
Comment (fofr) [inline app/posts/manage-teacher-training-applications/2020-01-31-withdrawing-an-offer.md]:
```suggestion
- Application withdrawn
```

**Review by fofr — APPROVED** [review]
A couple of minor tweaks. 👍

**Comment by fofr** [comment]
The confirm withdrawal screenshot needs to show a reason too.

**Comment by adamsilver** [comment]
Made tickets for next step tickets.

**Pull request merged**